doc/MediaSigning.xml

```diff
@@ -0,0 +1,689 @@
+              <?oxy_comment_end?></para>
+          </listitem>
+          <listitem>
+            <para>Serial Number or similar</para>
+          </listitem>
+          <listitem>
+            <para>Manufacturer (who is the signer)</para>
+          </listitem>
+        </itemizedlist>
+      </section>
+      <section>
+        <title>Hash list</title>
+        <para>This mandatory field contains the hash list, for a complete GOP
+          <?oxy_comment_start author="FredrikS" timestamp="20220922T123227+0200" comment="Replace with IDR frame. Sync with Streaming Spec"?>I<?oxy_comment_end?>
+          through
+          <?oxy_comment_start author="FredrikS" timestamp="20220920T130135+0200" comment="Include B-frames as well. Replace with intermediate frames?"?>P-frames<?oxy_comment_end?>
+          and the linked I-frame of the succeeding GOP.</para>
+        <itemizedlist>
+          <listitem>
+            <para>Tag version (1 byte)</para>
+          </listitem>
+          <listitem>
+            <para>All hashes in one big chunk (variable bytes)</para>
+          </listitem>
+        </itemizedlist>
+      </section>
+      <section>
+        <title>Signature</title>
+        <para>This mandatory field contains the Signature of the document.</para>
+        <itemizedlist>
+          <listitem>
+            <para>Tag version (1 byte)</para>
+          </listitem>
+          <listitem>
+            <para>Message digest hash selected from a list of supported hashing algorithms (1 byte)</para>
+          </listitem>
+          <listitem>
+            <para><?oxy_comment_start author="FredrikS" timestamp="20220920T130305+0200" comment="Add reason why this is needed"?>Actual
+              size of signature<?oxy_comment_end?> (2 bytes)</para>
+            <para>Not all signing algorithms generate a fixed size.</para>
+          </listitem>
+          <listitem>
+            <para>The signature (max signature size bytes)</para>
+            <para>A fixed size is reserved to be able to determine the payload size of the SEI.
+              This is necessary to be able to hash the SEI and generate the message digest.</para>
+          </listitem>
+        </itemizedlist>
+      </section>
+      <section>
+        <title>Arbitrary data</title>
+        <para>This optional field contains Abritrary data that the device manifacturer might want included.
+        There is no guarantee that the Arbitrary data is reasonable or readable by anyone not priviously
+        knowledgable in the area.</para>
+      </section>
+      <section>
+        <title>Certificate</title>
+        <para>This optional field contains the ceritficate corresponding to the public key. This field is optional
+        in order to not put undue bitrate burden on the videostream as x509 certificates can be quite large, especially if 
+        intermediate certificates also are included.</para>
```

Comment:
"intermediate certificates also are included" suggestion "intermediate certificates are included" removal of the "also" makes the overall paragraph more readable.

Reply:
Fixed